### PR TITLE
Azure Pipelines - Filter the build job on the destination side and reuse the type.detail field for the build stage name

### PR DESCRIPTION
**Pull request by sowa705**
## Description

We want to store the build stage with the build step

## Type of change
- [ ] Bug fix
- [x] New feature
- [x] Breaking change

### CODE REVIEW

**Comment by sowa705** [comment]
I have read the CLA Document and I hereby sign the CLA

File: destinations/airbyte-faros-destination/src/converters/azurepipeline/builds.ts [hunk 1]
```diff
@@ -67,11 +67,21 @@ export class Builds extends AzurePipelineConverter {
...
+      }
       const jobCreatedAt = Utils.toDate(job.startTime);
       const jobStartedAt = Utils.toDate(job.startTime);
       const jobEndedAt = Utils.toDate(job.finishTime);
       const jobStatus = this.convertBuildStepState(job.result);
-      const jobType = this.convertBuildStepType(job.type);
+
+      // get the parent job
+      let parentJob = job;
+      while (parentJob.parentId)
```
Comment (tovbinm) [inline destinations/airbyte-faros-destination/src/converters/azurepipeline/builds.ts:80]:
```
if (parentJob.parentId) {
        parentJob = build.jobs.find((j) => j.id == parentJob.parentId);
}
```

File: destinations/airbyte-faros-destination/src/converters/azurepipeline/builds.ts [hunk 2]
```diff
@@ -67,11 +67,21 @@ export class Builds extends AzurePipelineConverter {
     }
 
     for (const job of build.jobs) {
+      if (job.type != 'Job') {
+        continue;
+      }
```
Comment (tovbinm) [inline destinations/airbyte-faros-destination/src/converters/azurepipeline/builds.ts:72]: no other job types are interesting to save?

Comment (sowa705) [reply destinations/airbyte-faros-destination/src/converters/azurepipeline/builds.ts:72]: Saving individual tasks would be a bit too much i think

**Pull request closed without merging**